plugins/catalog-graphql/src/schema.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const EntityMetadataFields = /* GraphQL */ `
+  name: String!
+  annotations: JSONObject!
+  annotation(name: String!): JSON
+  labels: JSONObject!
+  label(name: String!): JSON
+  uid: String!
+  etag: String!
+  generation: Int!
+`;
+
+const EntityMetadata = /* GraphQL */ `
```

Comment:
hm this variable should probably just be called `schema` or something generic like that